lib/src/usage_impl_html.dart

```diff
@@ -25,6 +25,26 @@ Future<Analytics> createAnalytics(
   ));
 }
 
+class AnalyticsHtml extends AnalyticsImpl {
```

Comment:
I have reached in and used specific impl classes from usage clients before (`AnalyticsHtml`, ...) but am no longer sure that that is necessary. It's in a package private library `lib/src/`, so we can refactor to remove it later w/o changing the package version.